### PR TITLE
Created the 404 page

### PR DESCRIPTION
**Pull request by SergeyGV**
404card

### CODE REVIEW

File: UI/css/pageNotFound.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    max-width: 400px;
+    max-height: 400px;
+    padding-bottom: 50px;
+    padding-top: 50px;
+    width: auto;
+}
+
+#main-card {
+    max-width: 600px;
+    margin-left: auto;
```
Comment (esteeroramy) [inline UI/css/pageNotFound.css]: this can combined into `margin: auto`

File: UI/css/pageNotFound.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    margin-top: auto;
+    margin-bottom: auto;
+    text-align: center;
+}
+
+#card-container {
+    background-color: lightgray;
+}
+
+body {
```
Comment (esteeroramy) [inline UI/css/pageNotFound.css]: let's change this into a class

File: locales/en.json [hunk 3]
```diff
@@ -21,5 +21,9 @@
 	"userInformation": {
 		"password": "Password",
 		"username": "Username"
+	},
+	"pageNotFound": {
```
Comment (esteeroramy) [inline locales/en.json]: sort this alphabetically

File: views/pageNotFound.pug [hunk 4]
```diff
@@ -0,0 +1,19 @@
+extends layout
+
+block styles
+    link(rel='stylesheet' href='/css/pageNotFound.css')
+
+block title
+    title Velocity Page Not Found
```
Comment (esteeroramy) [inline views/pageNotFound.pug]: this should be in i18n

File: UI/css/pageNotFound.css (hunk 1 above)
Comment (SergeyGV) [reply UI/css/pageNotFound.css]: Thanks for the suggestion, combined it

File: UI/css/pageNotFound.css (hunk 2 above)
Comment (SergeyGV) [reply UI/css/pageNotFound.css]: What do you mean?

File: locales/en.json (hunk 3 above)
Comment (SergeyGV) [reply locales/en.json]: Done

File: UI/css/pageNotFound.css (hunk 2 above)
Comment (esteeroramy) [reply UI/css/pageNotFound.css]: so this is affecting the body, you can change this into like:
`.notFoundBody` and add that to the `.pug` file

File: views/pageNotFound.pug (hunk 4 above)
Comment (SergeyGV) [reply views/pageNotFound.pug]: Done

File: UI/css/pageNotFound.css (hunk 2 above)
Comment (SergeyGV) [reply UI/css/pageNotFound.css]: Ah, so the body itself isn't directly used, but with the same effect either way. Alright

**Pull request merged**